common/models/user.js

```diff
@@ -476,7 +476,10 @@ module.exports = function(User) {
             }
           });
         } else {
-          cb();
+          err = new Error("User '" + options.email + "' does not exist");
```

Comment:
The caller of this function should already have the email... Why put it in the error message?